src/script/lua_api/l_object.cpp

```diff
@@ -2046,7 +2046,8 @@ int ObjectRef::l_set_sky(lua_State *L)
 
 		lua_getfield(L, 2, "textures");
 		sky_params.textures.clear();
-		if (lua_istable(L, -1) && sky_params.type == "skybox") {
+		if (lua_istable(L, -1) && (sky_params.type == "skybox" ||
+			sky_params.type == "skybox_back" || sky_params.type == "skybox_front")) {
```

Comment:
```suggestion
				sky_params.type == "skybox_back" || sky_params.type == "skybox_front")) {
```

continuation lines are indented by two tabs so they don't blend in with statements on the next line. see https://dev.minetest.net/Code_style_guidelines. same for all the other occurrences.

but if you introduce the abstractions i suggested, you can probably get rid of all these two-line conditions.